py_zipkin/logging_helper.py

```diff
@@ -178,45 +160,34 @@ def _reset_queue(self):
         self.queue = []
         self.current_size = thrift.LIST_HEADER_SIZE
 
-    def add_span(
-        self,
-        span_id,
-        parent_span_id,
-        trace_id,
-        span_name,
-        annotations,
-        binary_annotations,
-        timestamp_s,
-        duration_s,
-        endpoint,
-        sa_endpoint,
-    ):
+    def add_span(self, internal_span):
+        span = internal_span.build_v1_span()
```

Comment:
nit: maybe call this span_builder?

Reply:
Well its job is to add the span to the queue, so the name is right.

Reply:
hah. that is true! I was actually talking about `internal_span`, not `add_span`, but it was ambiguous.

Reply:
Lol, @bplotnick ++. Changed it in https://github.com/Yelp/py_zipkin/pull/82